### PR TITLE
Setup deploy to different environments

### PR DESCRIPTION
**Pull request by rodrei**

### CODE REVIEW

**Comment by rodrei** [comment]
The configuration vars for each environment ideally should be in ENV vars, but for now this is a good enough solution I think.

File: bin/deploy.sh [hunk 1]
```diff
@@ -1,19 +1,33 @@
...
+  REACT_APP_API_URL=''
+elif [ "$ENV" == "staging" ]; then
+  BUCKET_NAME=member-services-staging.sumofus.org
+  WEBSITE_URL=https://member-services-staging.sumofus.org
+  CF_DISTRIBUTION_ID=E67T5UQJH9BML
+  REACT_APP_API_URL='https://qglxytlgti.execute-api.us-east-1.amazonaws.com/staging'
+else
+  echo "Don't know how to deploy ${ENV} environment"
+  exit 1
+fi
```
Comment (osahyoun) [inline bin/deploy.sh]: I'm tempted to suggest moving env vars to circleci, and killing the condition - instead we could have the vars prefixed with PRODUCTION_ and STAGING_. Anyway, looks good. I say merge!

Comment (rodrei) [reply bin/deploy.sh]: you'd still have to assign the preffixed vars  to the non-prefixed vars, like `BUCKET_NAME = $STAGING_BUCKET_NAME`, hence not being able to remove the condition. What we could do is use the same approach we're doing in api-services and use AWS Parameters Store. I'll merge for now and we can work on this later.

Comment (rodrei) [reply bin/deploy.sh]: Oh, just realized you could look for the prefixed var names inside the deploy script depending on the environment.

**Pull request merged**